README.md

```diff
@@ -1,58 +1,127 @@
-<img src="https://storage.googleapis.com/passage-docs/passage-logo-gradient.svg" alt="Passage logo" style="width:250px;"/>
+![passage-android](https://storage.googleapis.com/passage-docs/github-md-assets/passage-android.png)
 
-### Native passkey authentication for your Android app
-## Welcome!
-Integrating passkey technology can be really hard. That's why we built the Passage Android SDK - to make passkey authentication easy for you and your users, all in a native Android experience.
+[![Android](https://img.shields.io/badge/Android-3DDC84?logo=android&logoColor=white)](#) [![Kotlin](https://img.shields.io/badge/Kotlin-%237F52FF.svg?logo=kotlin&logoColor=white)](#) ![GitHub License](https://img.shields.io/github/license/passageidentity/passage-android)
+![Static Badge](https://img.shields.io/badge/Built_by_1Password-grey?logo=1password)
 
-<img width="700" height="1100" alt="image" src="https://github.com/user-attachments/assets/674fd6e6-0330-42e2-b467-eada7709f509">
+## About
 
-<br>
-<br>
+[Passage by 1Password](https://1password.com/product/passage) unlocks the passwordless future with a simpler, more secure passkey authentication experience. Passage handles the complexities of the [WebAuthn API](https://blog.1password.com/what-is-webauthn/), and allows you to implement passkeys with ease.
 
-## Usage
+Use [Passkey Flex](https://docs.passage.id/flex) to add passkeys to an existing authentication experience.
+
+Use [Passkey Complete](https://docs.passage.id/complete) as a standalone passwordless auth solution.
+
+Use [Passkey Ready](https://docs.passage.id/passkey-ready) to determine if your users are ready for passkeys.
+
+### In passage-android
+
+Use passage-android to implement Passkey Complete into your Android application to authenticate requests and manage users.
+
+| Product                                                                                                                                  | Compatible |
+| ---------------------------------------------------------------------------------------------------------------------------------------- | ---------- |
+| ![Passkey Flex](https://storage.googleapis.com/passage-docs/github-md-assets/passage-passkey-flex-icon.png) Passkey **Flex**             | ✅         |
+| ![Passkey Complete](https://storage.googleapis.com/passage-docs/github-md-assets/passage-passkey-complete-icon.png) Passkey **Complete** | ✅         |
+| ![Passkey Ready](https://storage.googleapis.com/passage-docs/github-md-assets/passage-passkey-ready-icon.png) Passkey **Ready**          | ✅         |
+
+## Getting Started
+
+#### Check Prerequisites
+
+<p>
+ You'll need a free Passage account and a Passkey Complete app set up in <a href="https://console.passage.id/">Passage Console</a> to get started. <br />
+ <sub><a href="https://docs.passage.id/home#passage-console">Learn more about Passage Console →</a></sub>
+</p>
 
 ### Passkey Complete
```

Comment:
Like the way you broke these up! It looks like `Passkey Complete` is duplicated, though, and it appears twice in the README (in addition to a section for Flex and Authentikit)

Reply:
Good catch, thanks!